java/languages.antlr/manifest.mf

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
+OpenIDE-Module: org.netbeans.modules.languages.antlr
```

Comment:
antlr is its fourth iteration - so unless this is really support for all antlr version, I suggest to name this `org.netbeans.modules.languages.antlr4`

Reply:
Ok.